### PR TITLE
Add adis16480 support

### PR DESCRIPTION
**Pull request by trishaange01**
# Description

Add support for adis16480.

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Hardware: ADIS16480
* OS: Windows

# Documentation

If this is a new feature or example please mention or link any documentation. All new hardware interface classes require documentation.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have signed off all commits and they contain "Signed-off by: <insert name>"
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by trishaange01** [comment]
Hi @tfcollins. This is ready for review. Thanks :)

**Comment by trishaange01** [comment]
Hi @tfcollins. I already added the missing docs. Also, the emulated tests (TEST-Emulated-HW) failed last time so I updated the xml file in test/emu/devices but I'm not sure if it's the cause of the failure.

**Comment by trishaange01** [comment]
Hi @tfcollins. This PR is ready for review. Thanks! 😊

File: supported_parts.md [hunk 1]
```diff
@@ -122,6 +122,7 @@
 - ADG2128
 - ADIS16460
 - ADIS16475
+- ADIS16480
```
Comment (tfcollins) [inline supported_parts.md:125]: Please include these:
           "adis16375",
            "adis16480",
            "adis16485",
            "adis16488",
            "adis16490",
            "adis16495-1",
            "adis16495-2",
            "adis16495-3",
            "adis16497-1",
            "adis16497-2",
            "adis16497-3"

**Review by tfcollins — CHANGES_REQUESTED** [review]
Just an update on the supported parts

File: supported_parts.md (hunk 1 above)
Comment (trishaange01) [reply supported_parts.md:126]: Already updated the supported parts. Thanks! :)

**Pull request merged**